typescript/sdk/src/deploy/verify/ContractVerifier.ts

```diff
@@ -349,29 +343,57 @@ export class ContractVerifier {
     const metadata = this.multiProvider.tryGetChainMetadata(chain);
     const rpcUrl = metadata?.rpcUrls[0].http ?? '';
     if (rpcUrl.includes('localhost') || rpcUrl.includes('127.0.0.1')) {
-      verificationLogger.debug('Skipping verification for local endpoints');
+      verificationLogger.debug(
```

Comment:
nit: you add name and address 4 times in logging, could DRY this if you wanted

Reply:
totally skimmed on the use of `const verificationLogger = logger.child({ chain, name: input.name,...`
added address as well so we're good to go